### PR TITLE
Add support for ARM

### LINKED ISSUE #1949: Support for ARM nodes
The A1 arm nodes are in [developer preview](https://github.com/aws/containers-roadmap/tree/master/preview-programs/). We would like to add the ability for `eksctl` to provision clusters with ARM nodes. 

At the moment there is a manual set of steps you need to achieve this: https://github.com/aws/containers-roadmap/tree/master/preview-programs/eks-arm-preview

More about Amazon A1 instances: https://aws.amazon.com/ec2/instance-types/a1/\

Linked ticket on AWS Container Roadmap: https://github.com/aws/containers-roadmap/issues/605

### PR DESCRIPTION
**Pull request by martina-if**
Thi PRs prepares eksctl for better handling clusters with ARM nodes.

closes #1949

- [ ] Add documentation

- [x] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [x] Manually tested
- [ ] Added labels for change area (e.g. `area/nodegroup`), target version (e.g. `version/0.12.0`) and kind (e.g. `kind/improvement`)
- [x] Make sure the title of the PR is a good description that can go into the release notes

### CODE REVIEW

**Comment by michaelbeaumont** [comment]
needs a rebase on `release-0.26` I think

**Comment by martina-if** [comment]
closes #1949

File: pkg/addons/default/addons.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+import (
+	"github.com/pkg/errors"
+
+	"github.com/weaveworks/eksctl/pkg/kubernetes"
+)
+
+// EnsureAllAddonsUpToDate checks the default addons (aws-node, kube-proxy and coredns) are up to date
+// and if they are not it will update them to the latest version for the given control plane version
+// TODO Delete this function when the multi architecture images are used by default when a new cluster is created. When
```
Comment (michaelbeaumont) [inline pkg/addons/default/addons.go:11]: can we track this as an issue?

Comment (martina-if) [reply pkg/addons/default/addons.go:11]: Good point: #2538

**Pull request merged**